frontend/src/queries/nodes/InsightViz/InsightVizDisplay.tsx

```diff
@@ -265,12 +270,13 @@ export function InsightVizDisplay({
                                     </div>
                                 </>
                             ) : (
-                                renderActiveView()
+                                <>{renderActiveView()}</>
                             )}
                         </div>
                     </>
                 )}
             </div>
+            <ResultCustomizationsModal />
```

Comment:
should we only show this in Edit mode? since the changes made outside it won't persist

Reply:
I opted to not do so, since this is also happening for other settings. We should address that in a proper "edit mode" overhaul.